share/arno-iptables-firewall/environment

```diff
@@ -1024,6 +1024,32 @@ parse_rule()
     dhost="$dhost_dport"
     ;;
 
+  shosts:ANYHOST-dhosts)
+    left_rule=$(echo "$rule" |cut -s -d'>' -f1)
+    right_rule=$(echo "$rule" |cut -s -d'>' -f2)
+
+    shosts=$(get_hosts_ih "$left_rule" "$ANYHOST")
+    dhosts=$(get_hosts_hp "$right_rule")
+
+    if [ -z "$dhosts" ]; then
+      parse_rule_warning "$rule"
+      return 1
+    fi
+    ;;
+
+  shosts-dhosts:ANYHOST)
```

Comment:
@arnova : This `shosts-dhosts:ANYHOST` does not appear to be used.

Reply:
Correct: It's for consistency and possible future use. It prevents breaking API when it's required by some plugin at some point in time.

Reply:
OK, fair enough.